scripts/licenses/main.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"bitbucket.org/creachadair/shell"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	confidenceThreshold float64
+	buildTags           string
+
+	rootCmd = &cobra.Command{
+		Use: "licenses",
+	}
+)
+
+func init() {
+	rootCmd.PersistentFlags().Float64Var(&confidenceThreshold, "confidence_threshold", 0.9, "Minimum confidence required in order to positively identify a license.")
+	// Go build flags, which should match flags offered by `go build`
+	rootCmd.PersistentFlags().StringVar(&buildTags, "tags", "", "A space-separated list of build tags to consider satisfied.")
```

Comment:
Tell that to the Go team!
```
$ go help build
...
-tags 'tag list'
	a space-separated list of build tags to consider satisfied during the
	build.
```
I intentionally made this identical to the flag from `go build` so that it could use $GOFLAGS.